.eslintrc.js

```diff
@@ -0,0 +1,82 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
+		es6: true,
+		node: true,
+		jest: true
+	},
+	globals: {
+		t: true,
+		n: true,
+		OC: true,
+		OCA: true,
+		'$': true,
+		'_': true,
+		Vue: true,
```

Comment:
You don't have any vue files?

Reply:
Not yet, as I wanted to do the migration in smaller chunks 😉

Reply:
Ah ok, so let's leave this I guess then :)